grafana/grafana
# InfluxDB: Run queries in parallel behind influxdbRunQueriesInParallel feature toggle

**What is this feature?**

Run multiple `influxql` queries in parallel instead of sequentially. 

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

InfluxDB InfluxQL query language users

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/grafana/issues/80955

**Special notes for your reviewer:**

### How to test
- Create a panel with many queries in it. 
- Save it and refresh it. 
- Panel queries should be run without error as usual

Please check that:
